Editor/Window/RegisterComputeInput.cs

```diff
@@ -302,6 +266,7 @@ private void LocalizeText()
             var l = new ElementLocalizer(_container);
             l.SetElementText("AnywherePageComputeNameLabel", Strings.AnywherePageComputeNameLabel);
             l.SetElementText("AnywherePageComputeIPLabel", Strings.AnywherePageComputeIPLabel);
+            l.SetElementText("AnywherePageComputeIPDescription", Strings.AnywherePageComputeIPDescription);
```

Comment:
We should use `defaultIpAddress` here instead. We might need to change this a little since this method wraps TextProvider right now